gui/packages/ubuntupro/lib/pages/subscription_status/subscription_status_widgets.dart

```diff
@@ -47,17 +47,18 @@ class SubscriptionStatus extends StatelessWidget {
         Container(
           padding: const EdgeInsets.all(16.0),
           decoration: BoxDecoration(
-            border: Border.all(color: const Color(0xFF0E8420), width: 1.0),
-            color: const Color(0xFFE6F8E8),
+            border: Border.all(color: theme.colorScheme.success, width: 1.0),
+            color: theme.colorScheme.success
+                .copyWith(lightness: 0.94, saturation: 0.56),
```

Comment:
AFAIK `.success` is an extension method brought you by yaru.dart. So, there is no benefit in not explicitely depending on YaruColors instead.